### ISSUE TITLE
Drop iOS 7 support. Anyone mind?

### ISSUE THREAD

**Issue by rsattar** [clusterinc/ClusterPrePermissions#24]
With Xcode 7 no longer supporting iOS 7 simulators, I think the writing is on the wall for iOS 7 support. Also, since June 1, 2015, [we have been required to submit our apps with at least the iOS 8 SDK](https://developer.apple.com/news/?id=12172014b).

**Comment by confile** [comment]
bad idea still 12% use ios7

**Comment by bizibizi** [comment]
Good idea. Also you need to customize 'Grant/No Grand' buttons. Currently 'No Grand' is bold, that's not good. We need to highlight  'Grant' button of alert

**Comment by rsattar** [comment]
I haven't found time to work on this lately, but have been meaning to get back into it and make some updates.

I think at this point, dropping iOS 7 makes sense. However, as fun as Swift is, it's NOT a good idea to be writing a library in Swift. First, you have to deal with the language changing (from 2.3 to 3.0 to 4.0), which will cause a lot of headaches as a library developer (do I maintain branches of the code for each version of Swift?). Additionally, it requires that the app using the library has to now ship with the swift runtime. Until Swift stabilizes its ABI (and ships the runtime as part of the OS, so we don't have to include it), it's an unnecessary burden that developers have to put up with.
